.gitignore

```diff
@@ -29,11 +29,15 @@ test/xml-results
 .idea
 cmake/projects/
 
+# Build and release products
+cmake/build/
+release/
+
 bin/api-reference-parser/vendor
 docs/api-reference
 /moai-sdk/
-/lib/

```

Comment:
Lib should be ignored. They are the output of the bin/build scripts
On 20 Oct 2015 8:39 am, "seclorum" <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/moai/moai-dev/pull/1144#discussion_r42435298>:
>
> >  bin/api-reference-parser/vendor
> >  docs/api-reference
> >  /moai-sdk/
> > -/lib/
>
> I don't remember why I did this. Can someone confirm if this should be
> gitignored?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/moai/moai-dev/pull/1144/files#r42435298>.
>